lang/nl.js

```diff
@@ -1,55 +1,59 @@
-	"components.form-element.input.number.rangeOverflow": "Getal moet {maxExclusive, select, true {less than} false {less than or equal to}} {max} zijn.",
-	"components.form-element.input.number.rangeUnderflow": "Getal moet {minExclusive, select, true {greater than} false {greater than or equal to}} {min} zijn.",
-	"components.form-element.input.text.tooShort": "{label} moet ten minste {minlength} tekens bevatten",
-	"components.form-element.input.url.typeMismatch": "URL is niet geldig",
-	"components.form-element.valueMissing": "{label} is vereist.",
-	"components.form-error-summary.errorSummary": "Er {count, plural, one {is 1 fout} other {zijn {count} fouten}} gevonden in de informatie die u hebt ingediend",
-	"components.input-date-range.endDate": "Einddatum",
-	"components.input-date-range.errorBadInput": "{startLabel} moet voor {endLabel} liggen",
-	"components.input-date-range.startDate": "Startdatum",
-	"components.input-date-time-range-to.to": "tot",
-	"components.input-date-time-range.endDate": "Einddatum",
-	"components.input-date-time-range.errorBadInput": "{startLabel} moet voor {endLabel} liggen",
-	"components.input-date-time-range.startDate": "Startdatum",
-	"components.input-date-time.date": "Datum",
-	"components.input-date-time.errorMaxDateOnly": "Datum moet voor {maxDate} liggen",
-	"components.input-date-time.errorMinDateOnly": "Datum moet na {minDate} liggen",
-	"components.input-date-time.errorOutsideRange": "Datum moet tussen {minDate} en {maxDate} liggen",
-	"components.input-date-time.time": "Tijd",
-	"components.input-date.clear": "Wissen",
-	"components.input-date.errorMaxDateOnly": "Datum moet voor {maxDate} liggen",
-	"components.input-date.errorMinDateOnly": "Datum moet na {minDate} liggen",
-	"components.input-date.errorOutsideRange": "Datum moet tussen {minDate} en {maxDate} liggen",
-	"components.input-date.openInstructions": "Gebruik datumnotatie {format}. Gebruik Pijl omlaag of druk op Enter om de mini-agenda te openen.",
-	"components.input-date.setToToday": "Instellen op Vandaag",
-	"components.input-search.clear": "Zoekopdracht wissen",
-	"components.input-search.search": "Zoeken",
-	"components.input-time-range.endTime": "Eindtijd",
-	"components.input-time-range.errorBadInput": "{startLabel} moet voor {endLabel} liggen",
-	"components.input-time-range.startTime": "Starttijd",
-	"components.list-item-drag-handle.default": "Itemactie voor {name} opnieuw rangschikken",
-	"components.list-item-drag-handle.keyboard": "Items opnieuw rangschikken, huidige positie {currentPosition} van {size}. Als u dit item wilt verplaatsen, drukt u op de pijl omhoog of omlaag.",
-	"components.menu-item-return.return": "Hiermee keert u terug naar het vorige menu.",
-	"components.menu-item-return.returnCurrentlyShowing": "Hiermee keert u terug naar het vorige menu. U bekijkt {menuTitle}.",
+	"components.alert.close": "$$$$$$$$$$$$$$$$ $$$$$$$$$",
```

Comment:
Uh... what happened here?

Reply:
Did we accidentally get the translations for Greenland instead?  That's a lot of green... 😛

Reply:
Oh... from Slack:
> We've had translators update their parsers to lock non-translatable text to avoid having to manually correct issues in PRs, usually in ICU message format strings
Could you check to make sure that all the $ characters represent translatable text in ICU strings, that everything else is correctly marked as non-translatable? (Note that translators can still add or move variables if needed)

> we're just looking to review the changes in the NL file

Reply:
I had seen that message but didn't realized it would result in this.